src/main/java/auth/TdpRecruitmentAuthenticator.java

```diff
@@ -20,16 +22,23 @@
 
 	private final TdpRecruitmentPasswordStore passwordStore;
 
-	private static final Cache<String, Person> cache = CacheBuilder
-			.newBuilder()
-			.maximumSize(100)
-			.expireAfterAccess(10, TimeUnit.MINUTES)
-			.build();
+	private final TdpRecruitmentApplicationConfiguration config;
```

Comment:
We don't need full app configuration here, `TdpRecruitmentCacheConfiguration` is enough.